packages/rateslib/meta.yaml

```
@@ -0,0 +1,23 @@
+package:
+  name: rateslib
+  version: 1.5.0
+  top-level:
+    - rateslib
+source:
+  url: https://files.pythonhosted.org/packages/02/d7/7695bb1b6a240ecfae77804d3fc242e30582fd83973f76dda03b23d921a6/rateslib-1.5.0.tar.gz
+  sha256: 7c7af2707f20974a70403689f84905757a4ed084e4c92229388d2fc30b331b04
+requirements:
+  run:
+    - numpy
+    - pandas
+    - matplotlib
+  executable:
+    - rustup
+about:
+  home: https://github.com/attack68/rateslib
+  PyPI: https://pypi.org/project/rateslib
+  summary: A fixed income library for trading interest rates
+  license: Creative Commons Attribution, Non-Commercial, No-Derivatives 4.0 International Licence
```

Could we standardise this better by using the [SPDX short-form identifier](https://spdx.org/licenses/CC-BY-NC-ND-4.0.html) for this license?
```suggestion
  license: CC-BY-NC-ND-4.0
```